### ISSUE TITLE
home page links

### ISSUE THREAD

**Issue by sergiocontrino** [ecrin-github/rms-portal-new#142]
on the dropdown menu

- [x] - substitute the the'data privacy policy' with 'Data Transfer Process', linked to https://crr.gitbook.io/crdsr/data-transfer-process
- [x] - move the link to crMDR to bottom place (and signal it is external?)

**Comment by sergiocontrino** [comment]
also:

- [ ] Canserv under GA 101058620, BY-COVID under GA 101046203
EOSC-Life under GA 824087 ->
GA 824087 (EOSC-Life), GA 101046203 (BY-COVID), GA 101058620 (Canserv)

**Comment by cudillal** [comment]
I didn't signal that the MDR link is external as technically all the other list items link to an external site (Gitbook)

**Comment by sergiocontrino** [comment]
true, but this will clarify things, so please add it (only mdr) when convenient. thanks!
not urgent.
also
- [  ] change text from `MetaData Repository crMDR` -> `MetaData Repository`
- [  ] replace the crMDR link in the banner with the `Data Sharing Policy`  one
and maybe
- [  ] add also RED with text `Regulatory and Ethics DB`
- [  ] we could use a separator for these 2 last items, possibly with a header like `Other Services from ECRIN`
